src/OMJulia.jl

```diff
@@ -122,14 +123,19 @@ mutable struct OMCSession
          args4=randstring(10)
       end
       if (Compat.Sys.iswindows())
-         @assert omc == nothing "A Custom omc path for windows is not supported"
-         omhome=ENV["OPENMODELICAHOME"]
-         ompath=replace(joinpath(omhome,"bin","omc.exe"),r"[/\\]+" => "/")
-         #ompath=joinpath(omhome,"bin")
-         #add omc to path if not exist
-         ENV["PATH"]=ENV["PATH"]*ompath
-         open(pipeline(`$ompath $args2 $args3$args4`))
-         portfile=join(["openmodelica.port.julia.",args4])
+         if (omc != nothing)
+           omhome=ENV["OPENMODELICAHOME"]
+           ompath=replace(omc,r"[/\\]+" => "/")
+           open(pipeline(`$ompath $args2 $args3$args4`))
+           portfile=join(["openmodelica.port.julia.",args4])
+         else
+            omhome=ENV["OPENMODELICAHOME"]
+            ompath=replace(joinpath(omhome,"bin","omc.exe"),r"[/\\]+" => "/")
+            #add omc to path if not exist
+            ENV["PATH"]=ENV["PATH"]*ompath
```

Comment:
This will modify the PATH for the Julia code and will append the path once for each omc that is called. Better would be to create a pipeline with PATH set only for the omc executable.
I'm also unsure why omc would need to set the path of itself. It sure isn't needed on Linux since we use absolute paths everywhere (and OPENMODELICAHOME is also not needed)...

Reply:
@sjoelund yes you are correct , appending path to the environment is not required. I will change this